README.md

```diff
@@ -44,33 +44,33 @@ Install the chart into a new namespace or an existing namespace as specified by
 helm upgrade -i monitoring-plugin charts/openshift-console-plugin -n my-plugin-namespace --create-namespace --set plugin.image=my-plugin-image-location
 ```
 
-## Local Development 
+## Local Development
 
-### Dependencies 
+### Dependencies
 1. [yarn](https://yarnpkg.com/en/docs/install)
-2. [oc](https://mirror.openshift.com/pub/openshift-v4/clients/oc/4.4/) 
-3. [podman 3.2.0+](https://podman.io) or [Docker](https://www.docker.com) 
+2. [oc](https://mirror.openshift.com/pub/openshift-v4/clients/oc/4.4/)
+3. [podman 3.2.0+](https://podman.io) or [Docker](https://www.docker.com)
 4. An OpenShift cluster
 
-### Running Locally 
+### Running Locally
 ```
-# Login to an OpenShift cluster 
+# Login to an OpenShift cluster
 $ oc login <clusterAddress> -u <username> -p <password>
 
-# Start podman (or Docker) 
-$ podman machine init 
-$ podman machine start 
+# Start podman (or Docker)
+$ podman machine init
+$ podman machine start
 
-# Install dependencies 
-$ make install 
+# Install dependencies
+$ make install
 
-# Run the application 
+# Run the application
 $ make start-frontend
-# In a separate terminal 
+# In a separate terminal
 $ make start-console
 ```
 The application will be running at [localhost:9000](http://localhost:9000/).
 
-### Local Development Troubleshooting 
+### Local Development Troubleshooting
 1. Disable cache. Select 'disable cache' in your browser's DevTools > Network > 'disable cache'. Or use private/incognito mode in your browser.
-
+2. Enable higher log verbosity by setting `-log-verbosity=1` when starting the plugin backend
```

Comment:
should this be `-log-level`?

Reply:
sure should be, I'll put a PR up to fix it later